src/Fantomas/FormatConfig.fs

```diff
@@ -116,7 +116,7 @@ type internal ColumnIndentedTextWriter(tw : TextWriter) =
         member __.Dispose() =
             indentWriter.Dispose()    
 
-type internal Context = 
+type Context = 
```

Comment:
It is removing `internal` really needed?

Reply:
No, my bad.